swagger.yml

```diff
@@ -1014,6 +1014,34 @@ paths:
             $ref: '#/definitions/plugin'
         default:
           $ref: '#/responses/error'
+  /sites/{site_id}/plugin_runs/latest:
+    get:
+      operationId: getLatestPluginRuns
+      tags: [x-internal]
+      description: 'This is an internal-only endpoint.'
+      parameters:
+        - name: site_id
+          in: path
+          type: string
+          required: true
+        - name: packages
```

Comment:
Does the call error when one of the `packages` has no `plugin_run` to return?

Reply:
No, it will just be omitted from the response. So, if no provided package has a `plugin_run` the response will be an empty array.

Reply:
Thanks!